### PR TITLE
internal/website: migrate CLI uploader tutorial

### PR DESCRIPTION
**Pull request by zombiezen**
READMEs updated to link to site.

Updates #1577

### CODE REVIEW

File: internal/website/content/tutorials/cli.md [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+concerns][] that the Go CDK makes possible. So, let's write some code!
+
+When we're done, our command line application will work like this:
+
+```shell
+# uploads gopher.png to GCS
+$ ./upload gs://go-cloud-bucket gopher.png
+
+# uploads gopher.png to S3
+$ AWS_REGION=us-west-1 ./upload s3://go-cloud-bucket gopher.png
```
Comment (eliben) [inline internal/website/content/tutorials/cli.md]: Is this absolutely necessary to specify explicitly? I somehow can run simple samples without it

File: internal/website/content/_index.md [hunk 2]
```diff
@@ -33,9 +33,10 @@ To start using the Go CDK, install it using `go get`:
 go get gocloud.dev
 ```
 
-Then follow the [Go CDK
-tutorial](https://github.com/google/go-cloud/tree/master/samples/tutorial).
-Links to additional documentation and samples are available below.
+Then follow the [Go CDK tutorial][]. Links to additional documentation and
+samples are available below and in the site navigation bar.
+
+[Go CDK tutorial]: {{< ref "/tutorials/cli.md" >}}
```
Comment (eliben) [inline internal/website/content/_index.md]: I'd suggest calling this tutorial `uploader` to be more in-line with the binary it builds and less tied to the notion of CLI which could be used for other providers etc.

File: internal/website/content/tutorials/cli.md (hunk 1 above)
Comment (zombiezen) [reply internal/website/content/tutorials/cli.md]: Ah yes, @vangent obviated this in #1418. Good catch.

File: internal/website/content/_index.md (hunk 2 above)
Comment (zombiezen) [reply internal/website/content/_index.md]: I'd like to add a more basic HTTP server tutorial at some point, and I was thinking the progression from command-line to HTTP would make more sense in terms of progression than "Uploader and Web Uploader". WDYT?

Comment (eliben) [reply internal/website/content/_index.md]: Fair enough, then maybe `cli-upload` or some such? `cli` could be also for pubsubs, CLIs and what not

Comment (zombiezen) [reply internal/website/content/_index.md]: Done.

**Pull request merged**